k8s/helm chart/argilla/templates/argilla-server-deployment.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: argilla-server-deployment
+  labels:
+    app: argilla-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: argilla-server
+  template:
+    metadata:
+      labels:
+        app: argilla-server
+    spec:
+      initContainers:
+        - name: wait-for-elasticsearch
+          image: alpine/curl:latest
+          command: [ "sh", "-c",
+            "ELASTICSEARCH_URL='http://elasticsearch:9200'; status_code=$(curl -s -o /dev/null -w '%{http_code}' $ELASTICSEARCH_URL);
+          while [ $status_code -ne 200 ]; do sleep 5; status_code=$(curl -s -o /dev/null -w '%{http_code}' $ELASTICSEARCH_URL);
+          echo Sleeping...; done; echo Elasticsearch Connected " ]
+      containers:
+        - name: argilla-server
+          image: argilla/argilla-server:latest
+          env:
+            - name: ARGILLA_ELASTICSEARCH
+              value: "http://elasticsearch:9200"

```

Comment:
can we make setting the value of `ELASTICSEARCH_URL` and `ARGILLA_ELASTICSEARCH` configurable?